### PR TITLE
enhance: Unit Test for Custom Tags

### PR DESCRIPTION
**Pull request by andytaylorkainos**

### CODE REVIEW

File: core/builtins/tests/test_custom_tags.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        span = '<span class="lite-highlight">'
+        span_end = '</span>'
+        actual_string = ct.highlight_text('this is a sassy string', 'a')
+        expected_string = 'this is ' + span + 'a' + span_end + ' s' + span + 'a' + span_end + 'ssy string'
+
+        self.assertEqual(actual_string, expected_string)
+
+    def test_strip_underscores(self):
+        expected_value = 'A long string'
+        actual_value = ct.strip_underscores('a_long_string')
```
Comment (markm5A8Z9F43913A) [inline core/builtins/tests/test_custom_tags.py]: Why did it lose capitalisation here

Comment (andytaylorkainos) [reply core/builtins/tests/test_custom_tags.py]: nono - it gained it

File: core/builtins/custom_tags.py [hunk 2]
```diff
@@ -84,7 +84,10 @@ def insert_str(string, str_to_insert, string_index):
     span = '<span class="lite-highlight">'
     span_end = '</span>'
 
+    loop = 0
     for index in indexes:
+        index += loop*(len(span) + len(term) + len(span_end) - 1)
```
Comment (markm5A8Z9F43913A) [inline core/builtins/custom_tags.py]: I think a comment would be useful here.

File: core/builtins/tests/test_custom_tags.py (hunk 1 above)
Comment (markm5A8Z9F43913A) [reply core/builtins/tests/test_custom_tags.py]: Edited my question :)

Comment (andytaylorkainos) [reply core/builtins/tests/test_custom_tags.py]: it's the expected functionality - the method is probably poorly named

**Pull request merged**